packages/ui/src/lib/Icon.svelte

```diff
@@ -1,4 +1,4 @@
-<script lang="ts" context="module">
+<script lang="ts" module>
```

Comment:
This breaks the build, you can't have two `<script>` unless one has `context="module"`!